engine/src/main/java/org/hibernate/search/engine/search/dsl/predicate/MatchPredicateFieldSetContext.java

```diff
@@ -69,13 +71,35 @@ default MatchPredicateFieldSetContext orRawField(String absoluteFieldPath) {
 
 	/**
 	 * Require at least one of the targeted fields to match the given value.
+	 * <p>
+	 * {@link DslConverter} option is {@code ENABLED} by default.
+	 * To change the option use {@link #matching(Object, DslConverter)} method instead.
 	 *
 	 * @param value The value to match.
 	 * The signature of this method defines this parameter as an {@link Object},
 	 * but a specific type is expected depending on the targeted field.
 	 * See <a href="SearchPredicateFactoryContext.html#commonconcepts-parametertype">there</a> for more information.
-	 * @return A context allowing to set options or get the resulting predicate.
+	 * @return A context allowing to set further options or get the resulting predicate.
+	 *
+	 * @see #matching(Object, DslConverter)
 	 */
-	MatchPredicateTerminalContext matching(Object value);
+	default MatchPredicateTerminalContext matching(Object value) {
+		return matching( value, DslConverter.ENABLED );
+	}
 
+	/**
+	 * Require at least one of the targeted fields to match the given value.
+	 *
+	 * @param value The value to match.
+	 * The signature of this method defines this parameter as an {@link Object},
+	 * but a specific type is expected depending on the targeted field.
+	 * See <a href="SearchPredicateFactoryContext.html#commonconcepts-parametertype">there</a> for more information.
+	 * @param dslConverter If {@code DISABLED} the value to match will be passed directly to the backend, no conversion will be applied.
+	 * See <a href="SearchPredicateFactoryContext.html#commonconcepts-rawfield">there</a> for more information about raw fields.
```

Comment:
Suggestion: replace this:

```
	 * @param dslConverter If {@code DISABLED} the value to match will be passed directly to the backend, no conversion will be applied.
	 * See <a href="SearchPredicateFactoryContext.html#commonconcepts-rawfield">there</a> for more information about raw fields.
```

with this:

```
	 * @param dslConverter Controls how the {@code value} should be converted before Hibernate Search attempts to interpret it as a field value.
	 * See {@link DslConverter}.
```

That way, we'll be able to move all the documentation related to DSL converters to... `DslConverter`.